utils.ts

```diff
@@ -73,7 +73,7 @@ export interface DsnResult {
 export function parseDsn(dsn: string): DsnResult {
   const url = new URL(dsn);
 
-  const params = {};
+  const params: Record<string, string> = {};
```

Comment:
btw, this whole `for` loop can be shortened by doing:
```
const params = Object.fromEntries(url.searchParams.entries());
```

Reply:
It could, but I didn't write that code and was trying to keep this pull as svelte as possible ;-)

Reply:
actually I think it's doing like a .clone() or something, I didn't want to break the preexisting code in case there's something going on that I didn't get.

Reply:
oh wait. you're right. but also it's bed time.